src/Database/DatabaseServiceProvider.php

```diff
@@ -54,6 +54,18 @@ public function register()
 
         $this->app->alias(ConnectionInterface::class, 'db.connection');
         $this->app->alias(ConnectionInterface::class, 'flarum.db');
+
+        $this->app->singleton('flarum.model.customRelations', function () {
+            return [];
+        });
```

Comment:
Out of interest: does `extend()` (in the extenders) also work if you use `$this->app->instance('...', [])` here?

Reply:
Did not know that existed, interesting! Not sure, can try to test though.

Reply:
No longer exists, so irrelevant